plugins/inventory/inventory.py

```diff
@@ -1420,6 +1425,8 @@ def parse(self, inventory, loader, path, cache=True):
         }
         if token:
             self.headers.update({"Authorization": "Token %s" % token})
+        if api_version:
+            self.headers.update({"Accept": f"application/json; version={api_version}"})
```

Comment:
@timizuoebideri1 Mind double checking this change? I pulled this change from pynautobot, but want to make sure the REST API inventory plugin supports versioning as well.